datasets/ru/egrul/crawler.py

```diff
@@ -605,14 +676,8 @@ def parse_examples(context: Context):
     # This subset contains a mix of companies with different address structures
     # and an example of successor/predecessor relationship
     for inn in [
-        "7709383684",
-        "7704667322",
-        "9710075695",
-        "7813654884",
-        "1122031001454",
-        "1025002029580",
-        "1131001011283",
-        "1088601000047",
+        "7714034350",
+        "7729348110",
```

Comment:
Let's keep the old ones around, but add the new ones - perhaps with a commend beside the new ones to indicate what's interesting about them